### ISSUE TITLE
Dockerize everything

### ISSUE THREAD

**Issue by dposse** [dposse/lets-go-out#4]
To be current with the devops world and such, we must dockerize the whole app!

Get to it Karsten 

:roll_eyes:

**Comment by krab7191** [comment]
Yeah... this is a bit out of scope. Something more apropos might be running postgres in a Docker container
